site/en/rules/deploying.md

```diff
@@ -81,6 +91,18 @@ In the following sections, assume the repository belongs to the
 workspace(name = "rules_mockascript")
 ```
 
+### MODULE.bazel *
+
+Refer to the [Bzlmod user guide](/docs/bzlmod#module-bazel) for detailed
+information on how rules can specify their dependencies using the `MODULE.bazel`
+file.
+
+Since the repository name under which a Bazel module is added can be freely
```

Comment:
I'm not so sure about that. What if there are multiple `rules_<?>`? I think the recommendation should be to use `<org>_rules_<lang>` also for `repo_name`.

We should probably also mention in this guide how to get the rules into BCR (i.e., linking to another doc)

Reply:
Since `repo_name` is chosen by end users, wouldn't a recommendation for it be a bit out of place in this doc? 

There is definitely some tension between "slick module names" and "prevent namegrabbing" that is probably best for the SIG to discuss and resolve. Maybe we can learn from other registries (NPM, cargo) here?